server/src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -211,8 +213,9 @@ public void init(boolean beginTransaction) throws Exception {
         locatorMap = this.injector.getInstance(ResourceLocatorMap.class);
```

Comment:
This ResourceLocatorMap instantiation is incorrect with the method locator being instantiated as it is below.

Reply:
Fixed.

Reply:
Fixed.

Reply:
Done.